code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -310,11 +310,17 @@
 		adjustHealth(unsuitable_atmos_damage)
 
 /mob/living/simple_animal/gib()
+	var/atom/Tsec
 	if(butcher_results)
-		var/atom/Tsec = drop_location()
+		Tsec = drop_location()
 		for(var/path in butcher_results)
 			for(var/i in 1 to butcher_results[path])
 				new path(Tsec)
+	if(guaranteed_butcher_results)
+		Tsec = drop_location()
+		for(var/path in guaranteed_butcher_results)
```

Comment:
for some cleaner code you could do
`for(var/path in guaranteed_butcher_results + butcher_results)`
And combine these two blocks

Reply:
Yeah thats nicer, thanks